<!---->
<launch>

<node pkg="joy" type="joy_node" name="joy_node" output="screen">
			<param name="joy_node/dev" value="/dev/input/js0"/>
	</node>

	<node pkg="gamepad" type="joystick_core" name="joy" output="screen">
			<param name="joy_sub" value="/joy"/>
			<param name="cmd_vel" value="/cmd_vel"/>
			<param name="cmd_vel_frame" value="/base_link"/>
	</node>

<node pkg="usb_relais" name="usb_relais" type="usb_relais" output="screen" respawn="true">
			<param name="relais_1" value="/joy/button_A" />	 
			<param name="relais_2" value="/joy/button_B" />
			<param name="relais_3" value="/joy/button_X" />	
			<param name="relais_4" value="/joy/button_Y" />		
		</node> 

<node pkg="usb_relais" name="time_activation_golf_ball" type="activate_relais">
			<param name="input_signal" value="/golf_ball" />	 
			<param name="waittime" value="2000" />	 
			<param name="output_signal" value="/joy/button_A" />	 
</node>	

<node pkg="usb_relais" name="time_activation_tennis_ball" type="activate_relais">
		 	<param name="input_signal" value="tennis_ball"/>	 
			<param name="waittime" value="5000" />	 
			<param name="output_signal" value="/joy/button_A" />	 
</node>
			

</launch>
